### PR TITLE
1141 false contributer

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1141. This is upbranched from #1156 so please merge after #1166

### CODE REVIEW

File: src/test/resources/alma/HT016709661.json [hunk 1]
```diff
@@ -129,6 +107,18 @@
...
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ctb",
+      "label" : "Mitwirkende"
+    },
+    "type" : [ "Contribution" ],
+    "agent" : {
+      "label" : "Düsseldorf",
+      "type" : [ "CorporateBody" ],
+      "id" : "https://d-nb.info/gnd/$1"
+    }
```
Comment (dr0i) [inline src/test/resources/alma/HT016709661.json:120]: Assuming this is the issue mentioned in https://github.com/hbz/lobid-resources/issues/1155, right?

**Review by dr0i — CHANGES_REQUESTED** [review]
Hm, as you have branched from another branch (not the master) and this other branch may change you have to incorporate that change also here. This may be complicated, and this is what I meant in today's standup with "better branch form the master and let's solve conflicts later".
Besides this: results look good +1 :)

File: src/test/resources/alma/HT016709661.json (hunk 1 above)
Comment (TobiasNx) [reply src/test/resources/alma/HT016709661.json:120]: yes

**Comment by TobiasNx** [comment]
I have set up a new branch and pr. #1170

**Pull request closed without merging**